lib/types.ts

```diff
@@ -32,29 +32,84 @@ export interface PluginMetadata {
   imageLayers: string[];
 }
 
-export interface Package {
+export interface PluginResponseStatic extends PluginResponse {
+  hashes: string[];
+}
+
+// NEW STUFF
+
+export interface PluginResponse {
+  plugin: PluginMetadata;
+  scannedProjects: ScannedProjectCustom[];
+}
+
+export interface ScannedProjectCustom {
```

Comment:
Maybe let's drop the custom ? In the cli the custom is:

![image](https://user-images.githubusercontent.com/16254901/80116292-b27a7f00-857d-11ea-873c-8c0f0f60fe1e.png)

Reply:
Another option is to move ScannedProjectCustom to cli-interface maybe and use it there ?

Reply:
that's what we have, except for the optional ```callGraph``` which I don't want to include yet, and for that here it's flattened.